FEM/rf_pcs.cpp

```diff
@@ -3371,6 +3375,10 @@ void CRFProcess::VariableDynamics()
 	pcs_secondary_function_name[pcs_number_of_secondary_nvals] = "STRAIN_PLS";
 	pcs_secondary_function_unit[pcs_number_of_secondary_nvals] = "-";
 	pcs_secondary_function_timelevel[pcs_number_of_secondary_nvals] = 1;
+	pcs_number_of_secondary_nvals++;
```

Comment:
In general, this way of adding secondary variable output is quite cumbersome and one has to make changes in many places. Christoph has implemented an easier way in ogs6. Maybe some inspiration can be gained there : )